src/iop/lut3d.c

```diff
@@ -378,6 +378,14 @@ uint8_t calculate_clut_haldclut(char *filepath, float **clut)
     return 0;
   }
   level *= level;  // to be equivalent to cube level
+  if(level > 256)
+  {
+    fprintf(stderr, "[lut3d] error - LUT 3D size %d > 256\n", level);
+    dt_control_log(_("error - LUT 3D size %d > 256"), level);
```

Comment:
This breaks translation... Maybe we can wait for 3.0.1. Also as this is a user's message I would just wrote:

   "Lut 3D size exceed the maximum supported"

The fprintf will tell what is the issue to developers.

Reply:
If that breaks translation only for that module this is maybe affordable ...
I'll make the changes.

Reply:
We could just protect dt from fault (for 3.0) and add proper messages later on (for 3.0.1). Let me know.

Reply:
Does adding new strings break the translation? Modifying - sure, but adding a new string will probably just mean that it will be left untranslated. If there is some technical problem with adding a new string, then we can just leave it non-translatable in 3.0 (remove _()) and revert it for 3.0.1, how about that?

Reply:
@parafin: adding a new string or changing it has the same result, it means that the string is not translated anymore. That is what I call "breaking translation", we promise a string freeze and we do not  fully do it. But this is better than a crash in this case, so will merge as soon it is stable.